### DISCUSSION TITLE
Google signing popup in Flutter

### DISCUSSION THREAD

**Discussion by nextdev1111** [supabase/supabase#7872] [Questions] [answered]
I am trying to login in supbase using Google, can you tell me how to implement popup in flutter instead of opening chrome.

**Comment by dshukertjr** [comment, marked as answer]
Hi @nextdev1111,

Signing in via popup is not yet supported in Supabase. You can follow [this issue](https://github.com/supabase-community/supabase-flutter/issues/5) to track the progress on it!

**Reply by nextdev1111** [reply]
When will it be implemented. I need it badly. Please provide it fast. Please 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏.
